### PR TITLE
Add templates for pull requests and issues

### PR DESCRIPTION
**Pull request by lornajane**
Let's make things clearer for contributors

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/question.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+name: ❓ Ask a question
+about: Got stuck or missing something from the docs? Ask away!
+---
+
+# What can we help you with?
+
+(for urgent or production platform issues, please contact the support team as normal)
```
Comment (ftisiot) [inline .github/ISSUE_TEMPLATE/question.md:8]: Should we add "via aiven.io" with a link to send people in the right direction?

**Review by ftisiot — CHANGES_REQUESTED** [review]
Adding a link in case of needing support could be useful.
Are the `request.md` and `review.md` swapped? 
e.g. shouldn't I use the `request` to ask for some content and the `review` to tell something is wrong?

**Comment by lornajane** [comment]
Some excellent points! Can you take another look please?

**Review by ftisiot — APPROVED** [review]
Seems all correct

**Pull request merged**